Sources/Architecture/X86/HardwareBreakpointManager.cpp

```diff
@@ -30,6 +31,65 @@ HardwareBreakpointManager::~HardwareBreakpointManager() {}
 
 int HardwareBreakpointManager::maxWatchpoints() { return kMaxHWStoppoints; }
 
+ErrorCode HardwareBreakpointManager::enableDebugCtrlReg(uint32_t &ctrlReg,
+                                                        int idx, Mode mode,
+                                                        int size) {
+  int enableIdx = 1 + (idx * 2);
+  int infoIdx = 16 + (idx * 4);
+
+  // Set G<idx> flag
+  ds2::Utils::EnableBit(ctrlReg, enableIdx);
```

Comment:
Don't hesitate to add `using ds2::Utils::EnableBit;` and `using ds2::Utils::DisableBit;` to this file instead of having to repeat the full namespace all over the place.